gallery/__manifest__.py

```diff
@@ -0,0 +1,81 @@
+{
+  'name': 'Gallery',
+  'version': '1.0',
+  'category': 'Retail',
+  'depends': [
+        'knowledge',
+        'l10n_be_pos_sale',
+        'partner_commission',
+        'pos_sale_subscription',
+        'project_sale_subscription',
+        'sale_purchase_project',
+        'sign',
+        'website',
```

Comment:
No need to add website I think as other modules like website_event_sale will have dependency of it.